wicket-jquery-ui-parent/wicket-jquery-ui-calendar/src/main/java/com/googlecode/wicket/jquery/ui/calendar6/resource/CalendarJavaScriptResourceReference.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.googlecode.wicket.jquery.ui.calendar6.resource;
+
+import de.agilecoders.wicket.webjars.request.resource.WebjarsJavaScriptResourceReference;
+
+/**
+ * Provides the resource reference for the fullcalendar javascript library.
+ *
+ * @author Sebastien Briquet - sebfz1
+ *
+ */
+public class CalendarJavaScriptResourceReference extends WebjarsJavaScriptResourceReference
+{
+	private static final long serialVersionUID = 1L;
+
+	private static class Holder {
```

Comment:
What is the benefit of the inner holder class vs. having the instance field be a static private final field of the outer resource reference class?